### DISCUSSION TITLE
Short-Term System Adequacy Report API

### DISCUSSION THREAD

**Discussion by aliati94** [ercot/api-specs#50] [Q&A]
Hi, I'm having trouble querying the short-term adequacy report (EMIL: np3-763-cd) through the API. It seems to me it's not available. Can someone confirm that? Are there plans to make it available?

**Comment by CaffeineLab** [comment]
I'm noticing that constraints reports (np6-86-cd and np4-191-cd) aren't returning data either.  So, systemwide?

**Comment by mackermann-ercot** [comment]
Hi aliati94, 

Currently the NP3-763-CD EMIL Product is not available through the Public API system; however, it is on the list of products to be added in the future.  I do not yet have a timeline.

Also, the two additional reports mentioned by CaffeineLab are currently available (see example URL's below).  I'm not sure why they would have been unavailable yesterday.  I did not see any interruptions in report uploads or errors in the Public API system.

https://api.ercot.com/api/public-reports/np6-86-cd/shdw_prices_bnd_trns_const?size=200
https://api.ercot.com/api/public-reports/np4-191-cd/dam_shadow_prices?size=200

Regards,
mackermann-ercot

**Reply by CaffeineLab** [reply]
There was a period where there were no reports being generated in for np6-86-cd...

![image](https://github.com/user-attachments/assets/9dfe1280-fbcf-43ef-8996-1eb8e8858017)

You can see the gap here.  

Now... there could have been no constraints within ERCOT during those 10 hours... but .....
